src/lxc/lxccontainer.c

```diff
@@ -4228,7 +4228,7 @@ static int do_lxcapi_snapshot(struct lxc_container *c, const char *commentfile)
 
 	strftime(buffer, 25, "%Y:%m:%d %H:%M:%S", &tm_info);
 
-	char *dfnam = alloca(strlen(snappath) + 1 + strlen(newname) + 1 + strlen(LXC_TIMESTAMP_FNAME) + 1);
+	dfnam = alloca(strlen(snappath) + 1 + strlen(newname) + 1 + strlen(LXC_TIMESTAMP_FNAME) + 1);
```

Comment:
Wait, how is that possible? There are no `alloca()` calls in master anymore... This is `must_realloc()` in master...

Reply:
Ok, Why are you changing this to `alloca()`? `alloca()` is banned from the codebase.